moduling/define.js

```diff
@@ -0,0 +1,15 @@
+//function checks if an input is an arrary or not
+const checkArray = (args) => {
+  if(Array.isArray(args)){
+    return args;
+  }else{
+    return `Invalid parameter type ${typeof args} supplied`;
+  }
+};
+
+
+// Export gives error (unexpected token) - 
```

Comment:
You are free to remove the comments as well as the commented code here. Since `module.exports` works, let us go with that for now.